features/guidance/turn.feature

```diff
@@ -940,12 +940,12 @@ Feature: Simple Turns
 
         And the ways
             | nodes | name          | highway   | oneway |
-            | ab    | Molkenmarkt   | secondary | yes    |
+            | ab    | Stralauer Str | secondary | yes    |
```

Comment:
Yep but `ab` is not at the intersection, it's the way right before `bc` which is Stralauer Str.

ab - Molkenmarkt - https://www.openstreetmap.org/way/23812656
bc - Stralauer Str - https://www.openstreetmap.org/way/286653195
cd - Stralauer Str - https://www.openstreetmap.org/way/60333309
ec - Molkenmarkt - https://www.openstreetmap.org/way/156364566